package.json

```diff
@@ -68,5 +77,11 @@
     "url": "https://github.com/kenwheeler/nuka-carousel/issues"
   },
   "homepage": "https://github.com/kenwheeler/nuka-carousel",
-  "sideEffects": false
+  "sideEffects": false,
+  "jest": {
+    "setupFiles": [
+      "<rootDir>/test/helpers/raf-polyfill.js",
+      "<rootDir>/test/helpers/jest-setup.js"
+    ]
```

Comment:
If you want to simplify setup you can use enzyme-matchers. new 5.0.0 version has a `jest-environment-enzyme` which greatly simplifies setup. 

* removes the requirement to install enzyme/adapter
* removes the requirement for your own enzyme adapter setup and raf polyfill

Reply:
Oh nice. I'll update this PR with that!